jalien.sh

```diff
@@ -0,0 +1,35 @@
+package: JAlien
+version: "%(tag_basename)s"
+tag: master
+source: https://gitlab.cern.ch/jalien/jalien.git
+requires:
+ - Java
+ - JAlien-XRootD
+build_requires:
+ - Java
+---
+#! /bin/bash -e
+
+rsync -av $SOURCEDIR/ ./
+./compile.sh users
+rsync -av . $INSTALLROOT/
+
+# Modulefile
+MODULEDIR="$INSTALLROOT/etc/modulefiles"
+MODULEFILE="$MODULEDIR/$PKGNAME"
+mkdir -p "$MODULEDIR"
+cat > "$MODULEFILE" <<EoF
+#%Module1.0
+proc ModulesHelp { } {
+  global version
+  puts stderr "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+}
+set version $PKGVERSION-@@PKGREVISION@$PKGHASH@@
+module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@@"
+# Dependencies
+module load BASE/1.0 ${JAVA_ROOT:+Java/$JAVA_VERSION-$JAVA_REVISION} ${JALIEN_XROOTD_ROOT:+JAlien-XRootD/$JALIEN_XROOTD_VERSION-$JALIEN_XROOTD_REVISION}
+# Our environment
+setenv JALIEN_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+prepend-path PATH \$::env(JALIEN_ROOT)
```

Comment:
Hmm this means that JAliEn installs executables _not_ under `bin`? If this is the case, it should be amended (by JAliEn authors). But we can also work it around within the recipe if this is not possible.

Reply:
Feedback from @costing is that after the compile command,
the only thing that needs to be installed is the alien-users.jar file,
and he seemed to have no preference as to it being put in the
JALIEN_ROOT dir, or a bin subdir

Reply:
We don't have any executables (yet at least). The command is built by the various services and boils down to something like: 
java -cp alien-users.jar Main

That said, we could organize it in a slightly different way:
- installation target (the only needed file so far): $JALIEN_ROOT/lib/alien-users.jar
- CLASSPATH += $JALIEN_ROOT/lib/alien-users.jar

That would make it even cleaner to run.

Reply:
JALIEN_ROOT might come handy if we want to build the command line in a different way. Please leave it in for now, we can remove it later if not needed.

Main is just an example / placeholder. The actual entry point will depend on the service, i.e. alien.site.JobAgent, alien.JSh, alien.JCentral ...

Reply:
Can't we simply introduce it when actually needed?

Reply:
Ok, let's try without, CLASSPATH should indeed be enough.